spring-integration-core/src/main/java/org/springframework/integration/config/IntegrationManagementConfiguration.java

```diff
@@ -64,12 +66,16 @@ public void setImportMetadata(AnnotationMetadata importMetadata) {
 
 	@Bean(name = IntegrationManagementConfigurer.MANAGEMENT_CONFIGURER_NAME)
 	@Role(BeanDefinition.ROLE_INFRASTRUCTURE)
-	public IntegrationManagementConfigurer managementConfigurer(ObjectProvider<MetricsCaptor> metricsCaptorProvider) {
+	public IntegrationManagementConfigurer managementConfigurer(
+			ObjectProvider<MetricsCaptor> metricsCaptorProvider,
+			ObjectProvider<ObservationRegistry> observationRegistryProvider) {
```

Comment:
Yes, that's what we have discussed with you a couple weeks ago. 
The conclusion was like: since a `micrometer-observation` is tiny API jar there is no reason in extra facade layer complexity on our side.
The bean by itself is `optional` though, so we just don't have any observation by default either way.